platform/security/filter/security-filter-web-sso/src/main/java/org/codice/ddf/security/filter/websso/WebSSOFilter.java

```diff
@@ -52,6 +56,10 @@
 
   ContextPolicyManager contextPolicyManager;
 
+  private String[] errorCodes = {
```

Comment:
❗️ This can/should be static.
✏️ My personal preference would be to be more verbose in declaring this by using `javax.ws.rs.core.ResponseStatus` or `org.apache.commons.httpclient.HttpStatus` enums, getting their values and converting to Strings. It's definitely more wordy but it is much more intentional.
❓ Are there any other response codes we need to handle?

Reply:
I referenced the codes from the web.xml in the Admin UI. The codes are 400-406 and 500-504